### PR TITLE
chore: use Superhero Wallet as app name

### LINKED ISSUE #3089: Update Mainfests to Stores - show superhero wallet instead of Superhero

### PR DESCRIPTION
**Pull request by martinkaintas**
closes #3089 

also removed unused translation entry

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Superhero",
+  "name": "Superhero Wallet",
   "description": "Superhero Wallet",
```
Comment (peronczyk) [inline src/manifest.json:3]: Maybe we can update this description to not be the same as the name? @onvisions

**Review by peronczyk — APPROVED** [review]

File: src/manifest.json (hunk 1 above)
Comment (onvisions) [reply src/manifest.json:3]: I have provided meta description suggestion long ago asked by @CedrikNikita.Actually we even have a backlog task for this one and it has been approved by @smaroudasunicorn. I guess it makes sense to use the same here. 
It was as follows: "Superhero is a multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Powered by æternity."

Comment (peronczyk) [reply src/manifest.json]: > Superhero is a multi-blockchain wallet...

But the name of the app is "Superhero Wallet". In other hand we would have this weird sentence "Superhero Wallet is a multi-blockchain wallet...". So maybe we could go without "Superhero is"? Ten the description would look as follows:

> Multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Powered by æternity.

@onvisions @smaroudas

Comment (onvisions) [reply src/manifest.json]: Sure. I agree.

**Review by smaroudasunicorn — APPROVED** [review]

File: src/manifest.json (hunk 1 above)
Comment (smaroudasunicorn) [reply src/manifest.json]: This PR is about changing Superhero to Superhero Wallet in the stores. From now on the brand name of our app is Superhero Wallet and not Superhero. We can create another task for welcome page changes.

File: src/manifest.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "Superhero",
-  "description": "Superhero Wallet",
+  "name": "Superhero Wallet",
+  "description": "Superhero is a multi-blockchain wallet to manage crypto assets and navigate the web3 and DeFi space. Powered by æternity.",
```
Comment (smaroudasunicorn) [inline src/manifest.json:3]: Superhero Wallet here

**Review by smaroudasunicorn — CHANGES_REQUESTED** [review]
Kindly update description

**Pull request merged**